examples/upload_files.py

```diff
@@ -22,25 +22,36 @@ async def get_files_to_upload(queue, path):
   """Finds which files will be uploaded to VirusTotal."""
   if os.path.isfile(path):
     await queue.put(path)
-    return
+    return 1
 
+  n_files = 0
   with os.scandir(path) as it:
     for entry in it:
         if not entry.name.startswith('.') and entry.is_file():
             await queue.put(entry.path)
+            n_files += 1
```

Comment:
This for loop is using indentation of 4 spaces instead of 2.

Reply:
Fixed at 444691846e1f5ef23e5d58ef3b69ba37d592d359